data/projects/enhance_coala_quickstart.md

```diff
@@ -13,43 +13,51 @@ However, there’s scope for improvement in the following areas:
   coala-quickstart. For example, if a project's Gulpfile.js specifies which
   files are to be linted with csslint, we could use this information to target
   CSSLintBear at exactly these files.
-- Improving the interface: 
+- Improving the interface:
 	- Removing non user-friendly warnings.
 	- Increasing automation.
 	- Validate user input before writing to the generated coafile.
-- Continuous Integration: Like other coala repositories, add Appveyor 
+- Continuous Integration: Like other coala repositories, add Appveyor
   (Windows) and Travis (OSX) integartions.
 
-In addition to the above changes the applicants are encouraged to come up with
-their own ideas.
-
 
 #### Milestones
 
-##### PREPARATION/BONDING
+ [BONDING] https://gitlab.com/coala/GSoC-2017/milestones/6
+ [PHASE 1] https://gitlab.com/coala/GSoC-2017/milestones/20
+ [PHASE 2] https://gitlab.com/coala/GSoC-2017/milestones/21
+ [PHASE 3] https://gitlab.com/coala/GSoC-2017/milestones/22
+ [STRETCH GOALS] https://gitlab.com/coala/GSoC-2017/milestones/41
+
+##### [PREPARATION/BONDING](https://gitlab.com/coala/GSoC-2017/milestones/6)
 
-* The applicant has identified all the kinds of configuration files to be worked
+* Identified all the kinds of configuration files to be worked
```

Comment:
well, whats a better way than starting it rather than doing it yourself :D ? @Nosferatul is doing it as well, you might just mention at the end: "Note: Those marked with * are ... " and then it would be an improvement brought by you, not something required by all projects.
you can do that in another commit if you wish, i dont mind.